### PR TITLE
Add description about user ATTRIBUTE

### PR DESCRIPTION
**Pull request by qiancai**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v6.4 (TiDB 6.4 versions)
- [ ] v6.3 (TiDB 6.3 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/11716
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: sql-statements/sql-statement-create-user.md [hunk 1]
```diff
@@ -73,6 +75,30 @@ CREATE USER 'newuser5'@'%' ACCOUNT LOCK;
...
+
+```sql
+CREATE USER 'newuser6'@'%' COMMENT 'This user is created only for test';
+```
+
+```
+Query OK, 1 row affected (0.02 sec)
+```
+
+Create a user with an `email` attribute:
```
Comment (qiancai) [inline sql-statements/sql-statement-create-user.md:90]:
```suggestion
Create a user with an `email` attribute.
```

File: information-schema/information-schema-user-attributes.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
+---
+title: USER_ATTRIBUTES
+summary: Learn the `USER_ATTRIBUTES` INFORMATION_SCHEMA table.
+---
+
+# USER_ATTRIBUTES
+
+The `USER_PRIVILEGES` table provides information about user comments and user attributes. This information comes from the `mysql.user` system table:
```
Comment (qiancai) [inline information-schema/information-schema-user-attributes.md:8]:
```suggestion
The `USER_PRIVILEGES` table provides information about user comments and user attributes. This information comes from the `mysql.user` system table.
```

File: information-schema/information-schema.md [hunk 3]
```diff
@@ -44,6 +44,7 @@ Many `INFORMATION_SCHEMA` tables have a corresponding `SHOW` command. The benefi
 | [`TABLE_CONSTRAINTS`](/information-schema/information-schema-table-constraints.md)      | Provides information on primary keys, unique indexes and foreign keys. |
 | `TABLE_PRIVILEGES`                                                                      | Not implemented by TiDB. Returns zero rows. |
 | `TRIGGERS`                                                                              | Not implemented by TiDB. Returns zero rows. |
+| [`USER_ATTRIBUTES`](/information-schema/information-schema-user-attributes.md) | Summarizes information about user comments and user attributes.
```
Comment (Oreoxmt) [inline information-schema/information-schema.md:47]:
```suggestion
| [`USER_ATTRIBUTES`](/information-schema/information-schema-user-attributes.md) | Summarizes information about user comments and user attributes. |
```

File: sql-statements/sql-statement-alter-user.md [hunk 4]
```diff
@@ -63,6 +65,30 @@ ALTER USER 'newuser' ACCOUNT LOCK;
...
+```
+
+```sql
+ALTER USER 'newuser' ATTRIBUTE '{"comment": null}';
+```
+
+```
+Query OK, 0 rows affected (0.02 sec)
+```
+
```
Comment (Oreoxmt) [inline sql-statements/sql-statement-alter-user.md:91]:
````suggestion
Modify the attributes of `newuser`:

```sql
ALTER USER 'newuser' ATTRIBUTE '{"newAttr": "value", "deprecatedAttr": null}';
SELECT * FROM information_schema.user_attributes;
```

```sql
+-----------+------+--------------------------+
| USER      | HOST | ATTRIBUTE                |
+-----------+------+--------------------------+
| newuser   | %    | {"newAttr": "value"}     |
+-----------+------+--------------------------+
1 rows in set (0.00 sec)
```

Modify the comment of `newuser` using `ALTER USER ... COMMENT`:

```sql
ALTER USER 'newuser' COMMENT 'Here is the comment';
SELECT * FROM information_schema.user_attributes;
```

```sql
+-----------+------+--------------------------------------------------------+
| USER      | HOST | ATTRIBUTE                                              |
+-----------+------+--------------------------------------------------------+
| newuser   | %    | {"comment": "Here is the comment", "newAttr": "value"} |
+-----------+------+--------------------------------------------------------+
1 rows in set (0.00 sec)
```

Modify the comment of `newuser` using `ALTER USER ... ATTRIBUTE`:

```sql
ALTER USER 'newuser' ATTRIBUTE '{"comment": null}';
SELECT * FROM information_schema.user_attributes;
```

```sql
+-----------+------+---------------------------+
| USER      | HOST | ATTRIBUTE                 |
+-----------+------+---------------------------+
| newuser   | %    | {"newAttr": "value"}      |
+-----------+------+---------------------------+
1 rows in set (0.00 sec)
```

````

File: sql-statements/sql-statement-create-user.md [hunk 5]
```diff
@@ -73,6 +75,30 @@ CREATE USER 'newuser5'@'%' ACCOUNT LOCK;
...
+{{< copyable "sql" >}}
+
+```sql
+CREATE USER 'newuser7'@'%' ATTRIBUTE '{"email": "user@pingcap.com"}';
+```
+
+```
+Query OK, 1 row affected (0.02 sec)
+```
+
```
Comment (Oreoxmt) [inline sql-statements/sql-statement-create-user.md:101]:
````suggestion
Create a user with a comment.

```sql
CREATE USER 'newuser6'@'%' COMMENT 'This user is created only for test';
```

```
Query OK, 1 row affected (0.02 sec)
```

Create a user with an `email` attribute.

```sql
CREATE USER 'newuser7'@'%' ATTRIBUTE '{"email": "user@pingcap.com"}';
```

```
Query OK, 1 row affected (0.02 sec)
```

````

**Review by Oreoxmt — COMMENTED** [review]
Rest LGTM

**Review by CbcWestwolf — APPROVED** [review]

File: sql-statements/sql-statement-alter-user.md [hunk 6]
```diff
@@ -63,6 +65,54 @@ ALTER USER 'newuser' ACCOUNT LOCK;
...
+```sql
++-----------+------+---------------------------+
+| USER      | HOST | ATTRIBUTE                 |
++-----------+------+---------------------------+
+| newuser   | %    | {"newAttr": "value"}      |
++-----------+------+---------------------------+
+1 rows in set (0.00 sec)
+```
+
```
Comment (Oreoxmt) [inline sql-statements/sql-statement-alter-user.md:115]: Remove L120-123 https://github.com/pingcap/docs-cn/pull/11716/commits/5d0808d35772928c862ff981ed4099f9b06f43b6

File: information-schema/information-schema-user-attributes.md [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+* `USER`: The user name.
+* `HOST`: The host from which the user can connect to TiDB. If the value of this field is `％`, it means that the user can connect to TiDB from any host.
+* `ATTRIBUTE`: The comment and attribute of the user, which are set by the [`CREATE USER`](/sql-statements/sql-statement-create-user.md) or [`ALTER USER`](/sql-statements/sql-statement-alter-user.md) statement.
+
+The following is an example:
+
+```sql
+CREATE USER testuser1 COMMENT 'This user is created only for test';
+CREATE USER testuser2 ATTRIBUTE '{"email": "user@pingcap.com"}';
+SELECT * FROM user_attributes;
```
Comment (Oreoxmt) [inline information-schema/information-schema-user-attributes.md:37]:
```suggestion
SELECT * FROM information_schema.user_attributes
```

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

File: information-schema/information-schema-user-attributes.md [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+* `USER`: The user name.
+* `HOST`: The host from which the user can connect to TiDB. If the value of this field is `％`, it means that the user can connect to TiDB from any host.
+* `ATTRIBUTE`: The comment and attribute of the user, which are set by the [`CREATE USER`](/sql-statements/sql-statement-create-user.md) or [`ALTER USER`](/sql-statements/sql-statement-alter-user.md) statement.
+
+The following is an example:
+
+```sql
+CREATE USER testuser1 COMMENT 'This user is created only for test';
+CREATE USER testuser2 ATTRIBUTE '{"email": "user@pingcap.com"}';
+SELECT * FROM information_schema.user_attributes
```
Comment (Oreoxmt) [inline information-schema/information-schema-user-attributes.md:37]:
```suggestion
SELECT * FROM information_schema.user_attributes;
```

**Review by Oreoxmt — APPROVED** [review]

**Pull request merged**